### PR TITLE
Resolves #22 [Doc] Better read me

### LINKED ISSUE #22: Better read me
Make a better read me with some syntax

README.md :
- [x] Explain the stack (back + front), add link to other readme in the project

server/README.md :
- [x] Add docker-compose cmd : Done here : 7cc6366
- [x] Move API documentation in doc folder

front/README.md : 
- [x] Add README file
- [x] Add python server cmd

### PR DESCRIPTION
**Pull request by yannis-mlgrn**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,24 @@
...
+
 Scratchy, application de tchat projet LPH.
 
+
+> 📂 Documentation directory is [here]("doc")
+
+
+## Backend
+
+ - 📂 Backend documentation's directory is [here]("doc/backend")
```
Comment (Benvii) [inline README.md:11]: No `"` needed

File: doc/frontend/frontend.md [hunk 2]
```diff
@@ -1,5 +1,15 @@
...
+  - [📂 **front directory**]("front")
+  
+  **How to start it :**
+
+  > run http server in front directory
+
+  ```bash
+  python3 -m http.server
+  ```
+
```
Comment (Benvii) [inline doc/frontend/frontend.md:12]: Add : Open webbrowser at http://localhost:5000

**Review by Benvii — CHANGES_REQUESTED** [review]

**Review by Draeggon — DISMISSED** [review]
there aren't the last update of the server/readme.md file in the new file doc/backend/API-documentation

**Review by Benvii — APPROVED** [review]

**Review by Benvii — DISMISSED** [review]

**Review by landerneRPi — CHANGES_REQUESTED** [review]
Récupère les modifs de Noam. pour insérer les commande curl PUT dans le server/README.md

**Review by landerneRPi — APPROVED** [review]

**Review by landerneRPi — APPROVED** [review]

**Review by landerneRPi — APPROVED** [review]

**Review by Draeggon — APPROVED** [review]

**Review by Benvii — COMMENTED** [review]
Changes are ok :+1:

**Pull request merged**